examples/example-vite-grpc-playground/src/components/ts-executor.ts

```
@@ -0,0 +1,24 @@
+            }
+            if (event.data.type === "error") {
+                reject(event.data.error);
+            }
+            worker.terminate();
+        };
+
+        worker.postMessage({ code });
+    });
+}
+
+export async function evaluateUserInput(query: string): Promise<any> {
```

Biome (1.9.4)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)